vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.h

```diff
@@ -148,6 +149,7 @@ struct WindowsTextInputComponentView
   HCURSOR m_hcursor{nullptr};
   std::chrono::steady_clock::time_point m_lastClickTime{};
   std::vector<facebook::react::CompWindowsTextInputSubmitKeyEventsStruct> m_submitKeyEvents;
+  std::string m_keyboardType{};
```

Comment:
We should not be storing copies of props on the component.  The value already exists in the prop object, lets not duplicate memory.